src/ProductAttribute.java

```diff
@@ -0,0 +1,4 @@
+public class ProductAttribute {
```

Comment:
let's mark this class as abstract, and derive attributes as concrete classes. like Size can be a String, Color can be a int, Images is a String or base64, etc.

Reply:
As i planned, size color ets would be objects of class ProductAttribute.
So, color object would be like
Productattribute color{
    name: "color",
    values: ["black","white","green"]
}

Reply:
color is good to be int (hex code) as we can assign this code in css on front-end
and if I want to expand the nuber of attributes you may need one big class that know everything about attributes, it's better to make every class single-responsible

Reply:
color is more convenient in int (hex value converted into decimal)

Reply:
and do not try to make one universal class, it is better for readability and debugging to make a little single-responsible classes